staging/src/k8s.io/metrics/pkg/client/clientset_test/clientset_test.go

```diff
@@ -40,3 +45,80 @@ func TestFakeList(t *testing.T) {
 		t.Errorf("Unexpected error: %v", err)
 	}
 }
+func TestFakeMetrics(t *testing.T) {
```

Comment:
in this test,if not fix the problem
list metrics will return emoty,then will errorf list pod metrics number is not match 5,0